### ISSUE TITLE
using instance IAM role

### ISSUE THREAD

**Issue by adilnaimi** [drone-plugins/drone-ecr#24]
build failed:

```
Drone AWS ECR Plugin built from c7b81ab
Please provide an access key id
[info] build failed (exit code 1)
```

my drone is running on an aws instance that has full access to ecr

can the pluging support IAM role ?

**Comment by AKoetsier** [comment]
See #16 for supporting IAM roles

**Comment by gjtempleton** [comment]
The changes to support this have now been merged into the 0.4 release.

@tboerger Think this can be closed with #26 having been merged.
